docs/acm/component/server-foundation/metrics.md

```diff
@@ -8,10 +8,11 @@ Nomenclature:
+| acm_managed_cluster_labels  | gauge  | Managed cluster labels | <table> <tr> <th>Number</th> <th>Label</th> <th>Description</th></tr> <tr> <td>1</td> <td>`hub_cluster_id`</td> <td>The cluster ID of the hub cluster.</td> </tr> <tr> <td>2</td> <td>`managed_cluster_id`</td> <td>The managed cluster ID is the cluster ID if the managed cluster is an OpenShift Container Platform cluster. Otherwise, the managed cluster ID is the cluster name.</td> </tr> </table> | STABLE |
+| acm_managed_cluster_info   | gauge | Managed cluster information | <table> <tr> <th>Number</th> <th>Label</th> <th>Description</th> </tr> <tr> <td>1</td> <td>`hub_cluster_id`</td> <td>The cluster ID of the hub cluster.</td> </tr> <tr> <td>2</td> <td>`managed_cluster_id`</td> <td>The managed cluster ID is the cluster ID if the managed cluster is an OpenShift Container Platform cluster. Otherwise, the managed cluster ID is the cluster name.</td> </tr> <tr> <td>3</td> <td>`vendor`</td> <td>The label `vendor` of the managed cluster.</td> </tr> <tr> <td>4</td> <td>`cloud`</td> <td>The label `cloud` of the managed cluster.</td> </tr> <tr> <td>5</td> <td>`service_name`</td> <td>The annotation `open-cluster-management/service-name` of the managed cluster. The values are `Compute` and `Other`.</td> </tr> <tr> <td>6</td> <td>`created_via`</td> <td>The annotation `open-cluster-management/created-via` updated by MCE. The supported values are `Discovery`,`AssistedInstaller`,`Hive` and `Other`.</td> </tr> <tr> <td>7</td> <td>`version`</td> <td>The OpenShift Container Platform version if the managed cluster is an OpenShift Container Platform cluster. Otherwise, the version is the Kubernetes version.</td> </tr> <tr> <td>8</td> <td>`available`</td> <td>The condition `ManagedClusterConditionAvailable` of the managed cluster. The values are `True` and `False`.</td> </tr> <tr> <td>9</td> <td>`core_worker`</td> <td>The core count of worker nodes on the managed cluster.</td> </tr> <tr> <td>10</td> <td>`socket_worker`</td> <td>The socket count of worker nodes on the {ocp-short} cluster.</td> </tr> </table> |  Not exposed, STABLE, New  |
```

Comment:
```suggestion
| acm_managed_cluster_info   | gauge | Managed cluster information | <table> <tr> <th>Number</th> <th>Label</th> <th>Description</th> </tr> <tr> <td>1</td> <td>`hub_cluster_id`</td> <td>The cluster ID of the hub cluster.</td> </tr> <tr> <td>2</td> <td>`managed_cluster_id`</td> <td>The managed cluster ID is the cluster ID if the managed cluster is an OpenShift Container Platform cluster. Otherwise, the managed cluster ID is the cluster name.</td> </tr> <tr> <td>3</td> <td>`vendor`</td> <td>The label `vendor` of the managed cluster.</td> </tr> <tr> <td>4</td> <td>`cloud`</td> <td>The label `cloud` of the managed cluster.</td> </tr> <tr> <td>5</td> <td>`service_name`</td> <td>The annotation `open-cluster-management/service-name` of the managed cluster. The values are `Compute` and `Other`.</td> </tr> <tr> <td>6</td> <td>`created_via`</td> <td>The annotation `open-cluster-management/created-via` updated by MCE. The supported values are `Discovery`, `AssistedInstaller`, `Hive`, and `Other`.</td> </tr> <tr> <td>7</td> <td>`version`</td> <td>The OpenShift Container Platform version if the managed cluster is an OpenShift Container Platform cluster. Otherwise, the version is the Kubernetes version.</td> </tr> <tr> <td>8</td> <td>`available`</td> <td>The condition `ManagedClusterConditionAvailable` of the managed cluster. The values are `True` and `False`.</td> </tr> <tr> <td>9</td> <td>`core_worker`</td> <td>The core count of worker nodes on the managed cluster.</td> </tr> <tr> <td>10</td> <td>`socket_worker`</td> <td>The socket count of worker nodes on the {ocp-short} cluster.</td> </tr> </table> |  Not exposed, STABLE, New  |
```